Update links in documentation (#6286)
PyCQA/pylint: doc/whatsnew/1.7.rst

  One of the major undertakings with Python 3 was a reorganization of the standard library to
  remove old or supplanted modules and reorganize some of the existing modules.  As a result,
  roughly 100 modules that exist in Python 2 no longer exist in Python 3.  See
  https://peps.python.org/pep-3108/ and https://peps.python.org/pep-0004/ for more

>   information.  For suggestions on how to handle this, see

Reviewer: ```suggestion
  information.  There were suggestions on how to handle this, at
```
